song_rater/views.py

```diff
@@ -1,11 +1,16 @@
 from django.shortcuts import render
+from song_rater.models import Song
+from django.views.generic.list import ListView
 
 
-# Create your views here.
 def index(request):
     context = {
         "title": "Welcome To The Ordina Music Rater",
         "text": "Make yourself at home"
-        }
+    }
 
     return render(request, 'song_rater/index.html', context)
+
+
+class SongListView(ListView):
+    model = Song
```

Comment:
This is where you could add a [`context_object_name`](https://docs.djangoproject.com/en/3.1/topics/class-based-views/generic-display/#making-friendly-template-contexts) attribute, as noted below in the comment attached to the template file.